src/index.html

```diff
@@ -2,13 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width,
+      user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="include">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header header__container">
+      <a href="#" class=" header__logo
+        logo">
+        <img class="logo__image" src="/src/images/logo.png"
+          alt="logo">
+      </a>
+      <nav class="header__menu">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link
+              header__link-blue header__link-apple
+              is-active">Apple</a>
+          </li>
```

Comment:
```suggestion
          <li class="header__item">
            <a href="#" class="header__link is-active">Apple</a>
          </li>
```
looks like redundant special classes for blue link and for what page this link.
for blue color should be responsible is-active link